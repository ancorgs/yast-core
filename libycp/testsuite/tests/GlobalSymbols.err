Parsed:
----------------------------------------------------------------------
`named_symbol
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
`_17
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
`_named_symbol
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  void `a = nil;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  boolean `b = true;
  return (b && false);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  integer `c = -2;
  return (c + 8);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  any `d = (0.00000000000000 - 2.00000000000000);
  return (d - 3.00000000000000);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  string `d = "composed";
  return (d + " string");
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  define func1(integer i) ``{
  y2error ("== local  func: %1", i);
}
;
  global define func1(integer i) ``{
  y2error ("== global func: %1", i);
}
;
  func1 (5);
  return 0;
}
----------------------------------------------------------------------
[Interpreter] tests/GlobalSymbols.ycp:42 Global symbol 'func1' shadows local symbol
[YCP] tests/GlobalSymbols.ycp(func1):34 == local  func: 5
Parsed:
----------------------------------------------------------------------
{
  define func2(integer i) ``{
  y2error ("== local  func: %1", i);
}
;
  if (true) {
  if (true) {
  global define func2(integer i) ``{
  y2error ("== global func: %1", i);
}
;
}
}
  func2 (5);
  return 0;
}
----------------------------------------------------------------------
[Interpreter] tests/GlobalSymbols.ycp:57 Global symbol 'func2' shadows local symbol
[YCP] tests/GlobalSymbols.ycp(func2):48 == local  func: 5
