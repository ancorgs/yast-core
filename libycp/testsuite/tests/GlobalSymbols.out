(`named_symbol)
(`_17)
(`_named_symbol)
(nil)
(false)
(6)
(-5.00000000000000)
("composed string")
(0)
(0)
